### PR TITLE
Add snapshot ARN as an option when creating an RDS instance

### PR DESCRIPTION
**Pull request by d-wierdsma**
#### Summary

Add snapshot ARN as an option when creating an RDS instance

#### Ticket Link

https://mattermost.atlassian.net/browse/CLD-3834

#### Release Note

```release-note
- Add snapshot ARN as an option when creating an RDS instance
```

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -36,8 +36,10 @@ func initCluster(apiRouter *mux.Router, context *Context) {
 //     "clusterID": "12345678",
 //     "backupRetentionPeriod": "15",
 //     "dbEngine: postgres",
-//     "replicas": "3"
-//     "dbProxy": true
+//     "maxConnections": "150000",
```
Comment (stylianosrigas) [inline api/api.go:39]: maxConnections are removed as an option

**Review by stylianosrigas — APPROVED** [review]
LGTM, just nit comment

File: api/api.go (hunk 1 above)
Comment (d-wierdsma) [reply api/api.go:39]: Sounds good

**Review by stylianosrigas — APPROVED** [review]
LGTM

**Pull request merged**